use-cases/manual-org-three-way.md

```diff
@@ -1,224 +1,228 @@
+
+
+  See the **account-management** and **account-logging** modules (`cloudtrail-sns` and `cloudtrail-s3` bucket) in the diagram given below. 
+
+- AWS member account for Sysdig (`SYSDIG_ACCOUNT_ID`)
+
+  - EKS or ECS cluster to deploy Sysdig Secure for Cloud (Cloud Connector)
+
+    See **account-management** and **account-security** modules in the diagram given below.
+
+- AWS member account for Compliance
+
+  - Sysdig Compliance Role:`aws:SecurityAudit policy`.  
+
+    For more information, see [Creating IAM roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html).
+
+    This IAM Role provisions permissions pertaining to enable audit trail for compliance. 
+
+    See the **account-compliance** module in the diagram given below.
+
+
+
+## Overview 
+
+In this setup, you will do the following:
+
+[three-way k8s setup](./resources/org-three-way-with-sns.png)
+
+-  AWS user account for logging:  In this account, you will create the Sysdig Access S3 Role,`SysdigS3AccessRole`, to retrieve data from the Cloudtail-enabled S3 bucket.
+
+- AWS user account for Sysdig: In this account, you will create the following:
+
+  - Sysdig Compute Role: `ARN_SYSDIG_COMPUTE_ROLE` as given in [Create Compute Role](#create-sysdig-compute-role). 
+
+    See the **account-security** modules in the diagram given below.
+
+  - A topic for `cloudtrail-sns-sqs` setting from the organizational Cloudtrail into Cloud Connector compute module. 
+
+We recommend that you perform the operations in the following order:
+
+1. Deploy the Cloud Connector. This configuration enables Threat Detection.
+2. Configure the Compliance role, `aws:SecurityAudit policy`,  if required.
+
+## Create Sysdig Compute Role
+
+To fetch data from the S3 bucket, you create a role named `SysdigComputeRole` and attach it to the compute service. Then, you download the ARN and save its as`ARN_SYSDIG_COMPUTE_ROLE`.
+
+1. Create `SysdigComputeRole` in your cluster. 
+
+2. Enable it to be used from within. 
+
+3. Do the following:
+
+   **EKS** cluster: Use the IAM authentication role mapping setup. 
+
+   ```json
+   {
+       "Version": "2012-10-17",
+       "Statement": [
+           {
+               "Effect": "Allow",
+               "Principal": {
+                   "Service": "eks.amazonaws.com"
+               },
+               "Action": "sts:AssumeRole"
+           }
+       ]
+   }
+   ```
+
+   
+
+   **ECS**: Allow Trust relationship for the ECS Task usage.
+
+   ```json
+   {
+       "Version": "2012-10-17",
+       "Statement": [
+           {
+               "Sid": "",
+               "Effect": "Allow",
+               "Principal": {
+                   "Service": "ecs-tasks.amazonaws.com"
+               },
+               "Action": "sts:AssumeRole"
+           }
+       ]
+   }
+   ```
+
+   For more information, see [Creating IAM roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html).
+
+4. Save the `ARN_SYSDIG_COMPUTE_ROLE`.
```

Comment:
Firstly, i propose homogenizing the document always using these two together
SysdigComputeRole (`ARN_SYSDIG_COMPUTE_ROLE`)

think we could simplify all this part just stating that
- When running the workload (EKS or ECS flavor), it must be done using the SysdigComputeRole (`ARN_SYSDIG_COMPUTE_ROLE`)

- This role can be configured to authenticate either
  - For Kubernetes cluster, [as Kubernetes Service account level](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html). see the `serviceAccount.*` properties of the cloud-connector chart we will use in further points
  - as ECS cluster, as the [TaskRole](https://docs.aws.amazon.com/AmazonECS/latest/userguide/task-iam-roles.html)

Reply:
where is the cloud-connector chart?